SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1817,7 +1817,7 @@ void USpatialReceiver::OnCommandRequest(const Worker_Op& Op)
 	UE_LOG(LogSpatialReceiver, Verbose, TEXT("Received command request (entity: %lld, component: %d, function: %s)"), EntityId, ComponentId,
 		   *Function->GetName());
 
-	RPCService->ProcessOrQueueIncomingRPC(ObjectRef, MoveTemp(Payload));
+	RPCService->ProcessOrQueueIncomingRPC(ObjectRef, MoveTemp(Payload), 0);
```

Comment:
I guess server RPCs should be ok as they use command and don't have any merging events?

Reply:
Do we trace commands? Saying that we probably don't need linear resolution for them if we do - but I think this would still emit an event (probably shouldn't since it doesn't match anywhere else)

Reply:
Yeah actually I am not sure if we track commands through the runtime... Perhaps something to look into later when it becomes relevant